release_info.json

```diff
@@ -0,0 +1,3 @@
+---
+---
```

Comment:
It might be worth adding a comment, so people aren't surprised about the front matter in the source file:

```suggestion
# This ensures that Jekyll reads this file
---
```